### PR TITLE
pbcopy,pbpaste: add Thai translations

### PR DESCRIPTION
**Pull request by tuterdust**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.th/osx/pbcopy.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# pbcopy
+
+> คัดลอกข้อมูลจากอินพุตมาตรฐาน (`stdin`) ไปยังคลิปบอร์ด ซึ่งเทียบได้กับการกดปุ่ม Cmd + C บนแป้นพิมพ์.
+> ข้อมูลเพิ่มเติม: <https://ss64.com/osx/pbcopy.html>.
+
+- คัดลอกเนื้อหาในไฟล์ที่กำหนดไปยังคลิปบอร์ด:
+
+`pbcopy < {{ที่อยู่ของไฟล์}}`
```
Comment (kbdharun) [inline pages.th/osx/pbcopy.md:8]:
```suggestion
`pbcopy < {{ทาง/ไป/ไฟล์}}`
```
This is what we use for this placeholder across all the pages. [Ref](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

File: pages.th/osx/pbcopy.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# pbcopy
+
+> คัดลอกข้อมูลจากอินพุตมาตรฐาน (`stdin`) ไปยังคลิปบอร์ด ซึ่งเทียบได้กับการกดปุ่ม Cmd + C บนแป้นพิมพ์.
```
Comment (kbdharun) [inline pages.th/osx/pbcopy.md:3]: `ซึ่งเทียบได้กับการกดปุ่ม Cmd + C บนแป้นพิมพ์`  (This is comparable to hitting Cmd + C on your keyboard.)

Can you add this part to the [English page](https://github.com/tldr-pages/tldr/blob/main/pages/osx/pbcopy.md) too? [As we want pages to be uniform across the original page and translations]

File: pages.th/osx/pbpaste.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# pbpaste
+
+> ส่งเนื้อหาของคลิปบอร์ดไปยังผลผลิตมาตรฐาน (`stdout`) ซึ่งเทียบได้กับการกดปุ่ม Cmd + V บนแป้นพิมพ์.
+> ข้อมูลเพิ่มเติม: <https://ss64.com/osx/pbpaste.html>.
+
+- เขียนเนื้อหาของคลิปบอร์ดไปยังไฟล์:
+
+`pbpaste > {{ที่อยู่ของไฟล์}}`
```
Comment (kbdharun) [inline pages.th/osx/pbpaste.md:8]:
```suggestion
`pbpaste > {{ทาง/ไป/ไฟล์}}`
```

File: pages.th/osx/pbpaste.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# pbpaste
+
+> ส่งเนื้อหาของคลิปบอร์ดไปยังผลผลิตมาตรฐาน (`stdout`) ซึ่งเทียบได้กับการกดปุ่ม Cmd + V บนแป้นพิมพ์.
```
Comment (kbdharun) [inline pages.th/osx/pbpaste.md:3]: `ซึ่งเทียบได้กับการกดปุ่ม Cmd + V บนแป้นพิมพ์.`  ( which is equivalent to hitting Cmd + V on your keyboard.)

Similarly, Can you add this to [English page](https://github.com/tldr-pages/tldr/blob/main/pages/osx/pbpaste.md) too?

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages.th/osx/pbcopy.md (hunk 2 above)
Comment (tuterdust) [reply pages.th/osx/pbcopy.md]: Sure, I already added it to the original English page

File: pages.th/osx/pbpaste.md (hunk 4 above)
Comment (tuterdust) [reply pages.th/osx/pbpaste.md]: Added it to the original English page

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**